src/hooks/useCourtSubscription.js

```diff
@@ -0,0 +1,17 @@
+import { useSubscription } from 'urql'
+import { useState } from 'react'
+import { CourtConfig } from '../queries/court'
+
+export default function useCourtSubscription() {
+  const [court, setCourt] = useState([])
+  const handleSubscription = (lastResponse = [], response) => {
+    return setCourt(response.courtConfig)
+  }
+  useSubscription(
+    {
+      query: CourtConfig,
+    },
+    handleSubscription
+  )
+  return court
+}
```

Comment:
Since we are not using the urql reducer, I think we could simplify this part (I haven’t tested!):

```jsx
 export default function useCourtSubscription() {
  const { data } = useSubscription({ query: CourtConfig })
  return data && data.courtConfig
}
```

Reply:
@fabriziovigevani you told me that you change the way we are doing the queries right? this file has been modified also?

Reply:
Yes, this is outdated. We will use a handler for the court config since we need to have some data converted. And we dont't need to useState for the data, we can just return the converted data in the handler.

What i did in a future next PR of the disputes interactions, is to grup all subscriptions in a single file `subscriptions-hooks` since the hook folder was getting bigger and made sense to group all hooks by matter, and don't have single hook files (same for queries). 

Do you think this makes sense?

Reply:
yes it makes sense for me, @bpierre do you have some thoughts about this?

Reply:
What @fabriziovigevani describes sounds great to me! Let’s resolve this for now then.